core/libs/commonwealth/commonwealth/settings/bases/pykson_base.py

```diff
@@ -92,7 +92,7 @@ def save(self, file_path: pathlib.Path) -> None:
 
         with open(file_path, "w", encoding="utf-8") as settings_file:
             logger.debug(f"Saving settings on: {file_path}")
-            settings_file.write(Pykson().to_json(self))
+            settings_file.write(json.dumps(json.loads(Pykson().to_json(self)), indent=4))
```

Comment:
Just checking, have you tried to use the `indent` parameter of [`Pykson::to_json()`](https://github.com/sinarezaei/pykson/blob/master/pykson/__init__.py#L1154)?

Reply:
Thanks! I'll update it.